.github/workflows/publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: Image Checks
+
+on:
+  push:
+    # Run only on release pushes
+    tags:
+      - v*
+  # but all PRs
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Generate Metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=semver,pattern={{version}}
+
+      - name: Build image
+        uses: docker/build-push-action@v6
+        env:
+          DOCKER_BUILD_RECORD_UPLOAD: false
+        with:
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha
+          tags: ${{ steps.meta.outputs.tags }}
+          load: true
+
+      - name: Login to GHCR
+        if: ${{ github.event_name == 'push' && github.ref_type == 'tag' }}
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io/${{ github.repository_owner }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish image
+        if: ${{ github.event_name == 'push' && github.ref_type == 'tag' }}
+        uses: docker/build-push-action@v6
+        env:
+          DOCKER_BUILD_RECORD_UPLOAD: false
+        with:
+          context: .
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
I wonder if this is the source of some of your woes?

ioc-adsimdetector does not use that and it gets its readme as a description by default and also uses a cache when pushing a tagged image.

Reply:
ioc-adsimdetector images don't get a descriptions either (eg [here](https://github.com/epics-containers/ioc-adsimdetector/pkgs/container/ioc-adsimdetector-developer/314680186?tag=2024.11.1). The readme is used for the top level package description for this repo with the current setup.

Reply:
I think we can resolve this conversation. It looks like you have the same 'features' as epics-containers and I'm not going to be looking at fixing those any time soon!